src/Fable.ReactGoogleMaps/Fable.Helpers.ReactGoogleMaps.fs

```diff
@@ -0,0 +1,172 @@
+module Fable.Helpers.ReactGoogleMaps
+
+open Fable.Core
+open Fable.Import
+open Fable.Core.JsInterop
+
+module Coordinates =
+
+    type [<Pojo>] Position = {
```

Comment:
Fable 2 doesn't do anything with the Pojo attribute any more. Seems React is not complaining now if you pass a non-plain JS object.